src/components/box/box.stories.mdx

```diff
@@ -205,23 +208,32 @@ Use the `scrollVariant` prop to indicate which theme scrollbar you would like to
   </Story>
 </Preview>
 
-<StyledSystemProps spacing color flexBox layout />
-<ArgsTable
-  rows={[
-    {
-      name: "overflowWrap",
-      type: { summary: "break-word | anywhere" },
-      description:
-        'String to set Box content break strategy. Note "anywhere" is not supported in Safari and IE11',
-      required: false,
-      defaultValue: "",
-    },
-    {
-      name: "scrollVariant",
-      type: { summary: "dark | light" },
-      description: "String to denote which style variant to use for scrollbars",
-      required: false,
-      defaultValue: "",
-    },
-  ]}
+## Props 
+
+### Style System
+
+<StyledSystemProps
+  spacing
+  color
+  flexBox
+  layout
+  noHeader
 />
+
+### Scroll Variant 
+
+<PropsTable rows={[
```

Comment:
This prop tables is not rendered, you forgot to import `PropsTable` at the top of this file